dluhc-component-library/src/components/timetableForm/TimetableForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { useMemo, useState } from "preact/hooks";
+import DescriptionPage from "./components/DescriptionPage";
+import PublishedDatePage from "./components/PublishedDatePage";
+import TitlePage from "./components/TitlePage";
+
+const TIMETABLE_FORM_PAGES = new Map([
+  ["title", TitlePage],
```

Comment:
If we do keep both variables for the pages, you should pull `title`, `description` and `publishedDate` out into constants so they always match.

Reply:
I think this isn't needed with the way I've done it as a list now, but if that changes and we do have both variables I'll make this change.